SUSE/rmt
# Update translations

## Description

card: https://trello.com/c/f5NGZ1Y3/2782-translation-round-finishing-touches

Integrate latest translations into RMT.

**How to review this pull request:**

- Checkout the branch and use RMT cli using `LANG=de_DE` or similar to change the language to desired test language
- Check if the translations are working fine (check updated translations show up in RMT)

# Do remember to **not** delete this branch after merge. This is the `i18n` branch!
